ebos/nncsorter.cpp

```diff
@@ -48,33 +53,49 @@ std::vector<Opm::NNCdata> sortNncAndApplyEditnnc(const std::vector<Opm::NNCdata>
                 return entry;
         };
 
+
+    auto format_ijk = [&cartDims](std::size_t cell) -> std::string {
+        auto i = cell % cartDims[0]; cell /= cartDims[0];
+        auto j = cell % cartDims[1];
+        auto k = cell / cartDims[1];
+
+        return fmt::format("({},{},{})", i + 1,j + 1,k + 1);
+    };
+
+
+    auto make_warning = [&format_ijk] (const Opm::KeywordLocation& location, const Opm::NNCdata& nnc) -> std::string {
+        return fmt::format("Problem with EDITNNC keyword\n"
+                           "In {} line {} \n"
```

Comment:
Tiny nit: Why the trailing blank before the newline?

Reply:
No reason - removed